product-service/src/handlers/createProduct/createProduct.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { createProduct } from './createProduct';
+
+const mockData = {
+  id: 'f48edfd3-adc4-4871-99c5-177a9eb2d0f3',
+  title: 'title',
+  description: 'description',
+  price: '10',
+  count: 10
+};
+
+
+jest.mock('src/utils/db/connect', () => ({
+  createConnection: async () => ({
+    query: async () => ({
+      rows: [mockData]
+    })
+  }),
+  closeConnection: () => { }
+}));
+
+describe('createProduct ', () => {
+  it('createProduct should return 400 status code with wrong body', async () => {
+    // @ts-ignore
```

Comment:
avoid ts-ignore. It's preferable to check types or cast it to something you need. Otherwise you can miss something not related to type mismatch that you expect.

Reply:
Yes, I avoid `// @ts-ignore` in development code, but I decided to leave `// @ts-ignore` in tests because of time saving. I would avoid `// @ts-ignore` in tests also in production code.